.storybook/preview.jsx

```diff
@@ -13,7 +14,9 @@ export const decorators = [
     <TestEnvironment>
       <ApolloWrapper>
         <ThemeProvider theme={theme}>
-          <Story />
+          <ErrorBoundary>
+            <Story />
+          </ErrorBoundary>
```

Comment:
I find it useful to wrap the story in an ErrorBoundary. This way you can still open a story that errors and see the error in the console. Otherwise storybook will omit the story from the menu altogether and pretend it doesn't exist.